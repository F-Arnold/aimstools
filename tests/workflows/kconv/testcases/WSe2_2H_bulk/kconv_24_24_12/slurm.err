/lustre/ssd/ws/kempt-aims_19596806
