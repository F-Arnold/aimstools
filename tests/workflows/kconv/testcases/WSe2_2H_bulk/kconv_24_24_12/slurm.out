Module aims/aims_200112 and 8 dependencies loaded.
Info: creating workspace.
remaining extensions  : 2
remaining time in days: 7
 *** trusted_descent: Too many uphill relaxation steps, overall above aggregated_energy_tolerance.
